packages/framework/src/types/step.types.ts

```diff
@@ -173,47 +173,62 @@ export type ChannelStep<
 ) => StepOutput<T_Result>;
 
 export type EmailOutput = FromSchema<(typeof channelStepSchemas)[ChannelStepEnum.EMAIL]['output']>;
+export type EmailOutputUnvalidated = FromSchemaUnvalidated<
+  (typeof channelStepSchemas)[ChannelStepEnum.EMAIL]['output']
+>;
 export type EmailResult = FromSchema<(typeof channelStepSchemas)[ChannelStepEnum.EMAIL]['result']>;
 
 export type SmsOutput = FromSchema<(typeof channelStepSchemas)[ChannelStepEnum.SMS]['output']>;
+export type SmsOutputUnvalidated = FromSchemaUnvalidated<(typeof channelStepSchemas)[ChannelStepEnum.SMS]['output']>;
 export type SmsResult = FromSchema<(typeof channelStepSchemas)[ChannelStepEnum.SMS]['result']>;
 
 export type PushOutput = FromSchema<(typeof channelStepSchemas)[ChannelStepEnum.PUSH]['output']>;
+export type PushOutputUnvalidated = FromSchemaUnvalidated<(typeof channelStepSchemas)[ChannelStepEnum.PUSH]['output']>;
 export type PushResult = FromSchema<(typeof channelStepSchemas)[ChannelStepEnum.PUSH]['result']>;
 
 export type ChatOutput = FromSchema<(typeof channelStepSchemas)[ChannelStepEnum.CHAT]['output']>;
+export type ChatOutputUnvalidated = FromSchemaUnvalidated<(typeof channelStepSchemas)[ChannelStepEnum.CHAT]['output']>;
 export type ChatResult = FromSchema<(typeof channelStepSchemas)[ChannelStepEnum.CHAT]['result']>;
 
 export type InAppOutput = FromSchema<(typeof channelStepSchemas)[ChannelStepEnum.IN_APP]['output']>;
+export type InAppOutputUnvalidated = FromSchemaUnvalidated<
+  (typeof channelStepSchemas)[ChannelStepEnum.IN_APP]['output']
+>;
 export type InAppResult = FromSchema<(typeof channelStepSchemas)[ChannelStepEnum.IN_APP]['result']>;
 
 export type DelayOutput = FromSchema<(typeof actionStepSchemas)[ActionStepEnum.DELAY]['output']>;
+export type DelayOutputUnvalidated = FromSchemaUnvalidated<(typeof actionStepSchemas)[ActionStepEnum.DELAY]['output']>;
 export type DelayResult = FromSchema<(typeof actionStepSchemas)[ActionStepEnum.DELAY]['result']>;
 
 export type digestRegularOutput = FromSchema<typeof digestRegularOutputSchema>;
+export type digestRegularOutputUnvalidated = FromSchemaUnvalidated<typeof digestRegularOutputSchema>;
 export type digestTimedOutput = FromSchema<typeof digestTimedOutputSchema>;
+export type digestTimedOutputUnvalidated = FromSchemaUnvalidated<typeof digestTimedOutputSchema>;
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Perhaps we can deprecate the old ones and fix capitalization of the types here?‏

Reply:
Thanks for highlighting this. I decided to update everything to have consistent capitlization. Deprecating is not needed as these uncapitlized types will almost certainly be used only by us.

In the rare case that any consumers are using the uncapitlized types, they will be greeted with a suggestion to use the new type:
<img width="699" alt="image" src="https://github.com/user-attachments/assets/357c4d16-2236-4b9a-b0cc-6d10bee87aa8">